api/src/team/application/api/certification-center.api.js

```diff
@@ -0,0 +1,11 @@
+import { usecases } from '../../domain/usecases/index.js';
+
+/**
+ * @function
+ * @param {number} certificationCenterId
+ * @param {date} archiveDate
+ * @param {number} archivedBy
+ */
```

Comment:
Correction de la syntax de la JSDoc pour un Object : 
```suggestion
/**
 * @function
 * @param {Object} params
 * @param {number} params.certificationCenterId
 * @param {date} params.archiveDate
 * @param {number} params.archivedBy
 */
```